embabel-agent-api/pom.xml

```diff
@@ -136,6 +136,11 @@
             <artifactId>spring-ai-ollama</artifactId>
         </dependency>
 
+        <dependency>
```

Comment:
Eventually we may want to split out the dependencies for different model providers, but for now this is good